src/main/java/com/sprint/mission/discodeit/controller/ChannelController.java

```diff
@@ -0,0 +1,77 @@
+package com.sprint.mission.discodeit.controller;
+
+import com.sprint.mission.discodeit.dto.data.ChannelDto;
+import com.sprint.mission.discodeit.dto.request.PrivateChannelCreateRequest;
+import com.sprint.mission.discodeit.dto.request.PublicChannelCreateRequest;
+import com.sprint.mission.discodeit.dto.request.PublicChannelUpdateRequest;
+import com.sprint.mission.discodeit.entity.Channel;
+import com.sprint.mission.discodeit.service.ChannelService;
+import com.sprint.mission.discodeit.util.LogMapUtil;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.RequiredArgsConstructor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/api/channels")
+@Tag(name = "Channel", description = "Channel Api")
+public class ChannelController {
+
+  private final ChannelService channelService;
+  private static final Logger log = LoggerFactory.getLogger(ChannelController.class);
+
+  @PostMapping("/public")
```

Comment:
요거는 단순화하려면 /public, /private으로 세부 path를 설정하는 것보다 쿼리 파라미터로 구분하면 좋습니다.

```suggestion
  POST /api/channels?type=PUBLIC
```